src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int INCREASE_SIZE = 2;
+    private int size;
+    private Node<K, V>[] buckets;
+
+    public MyHashMap() {
+        buckets = new Node[INITIAL_DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int threshold = (int) (INITIAL_DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR);
+        Node<K, V> node = new Node<>(key, value, null);
+        int index = hash(key);
+        if (size == threshold) {
```

Comment:
We can avoid this variable
And our threshold will always be the same since we do not change this constant INITIAL_DEFAULT_CAPACITY
```suggestion
        Node<K, V> node = new Node<>(key, value, null);
        int index = hash(key);
        if (size > buckets.length * DEFAULT_LOAD_FACTOR) {
```